docs/pages/guides/monorepos.mdx

```diff
@@ -282,6 +294,30 @@ export default function App() {
 
 As mentioned earlier, using monorepos is not for everyone. You take on increased complexity and need to solve issues you most likely will run into. Here are a couple of common issues you might encounter.
 
+### Multiple React Native versions within the monorepo
+
+Expo SDK 50 and higher has improved support for more complext `node_modules` patterns, such as isolated modules. Unfortunately, React Native itself can still cause issues when installing multiple versions inside a single monorepo. Because of that, it's recommended to only use a single version of React Native.
+
+You can check if your monorepo has multiple React Native versions, and why they are installed, through the package managers.
+
+<Terminal cmd={[
```

Comment:
I also think we should use Tabs to separate diff pkg manage commands here so that folks can still copy paste it (just a thought, but should block this PR).

Reply:
That would be great, especially here!

Reply:
I can follow up in another PR too as it should not be blocking this PR (sorry for the wrong context in my previous comment).

Reply:
No worries, let's do that!